notebooks/405-paddleOCR-webcam/405-paddleOCR-webcam.ipynb

```diff
@@ -0,0 +1,466 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e541150c-0f98-41c6-a97c-97acb26efd2f",
+   "metadata": {},
+   "source": [
+    "#### Load the Network for Text Detection"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "f9c5c83a-961c-4d98-8b20-5e96c8ef71f3",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "det_model_dir = \"./model/ch_ppocr_mobile_v2.0_det_infer\"\n",
+    "det_model_file_path = det_model_dir + \"/inference.pdmodel\"\n",
+    "det_params_file_path = det_model_dir + \"/inference.pdiparams\"\n",
+    "\n",
+    "# initialize inference engine for text detection\n",
+    "det_ie = Core()\n",
+    "det_net = det_ie.read_model(model=det_model_file_path, weights=det_params_file_path)\n",
+    "det_compiled_model = det_ie.compile_model(model=det_net, device_name=\"CPU\")\n",
+    "\n",
+    "# get input and output nodes for text detection\n",
+    "det_input_layer = next(iter(det_compiled_model.inputs))\n",
+    "det_output_layer = next(iter(det_compiled_model.outputs))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "5ec5c940-626c-4cf7-a90f-833200969846",
+   "metadata": {},
+   "source": [
+    "#### Load the Network for Text Recognition"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "89c0a07a-8186-47b5-ad95-f104a84d13d8",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "rec_model_dir = \"./model/ch_ppocr_mobile_v2.0_rec_infer\"\n",
+    "rec_model_file_path = rec_model_dir + \"/inference.pdmodel\"\n",
+    "rec_params_file_path = rec_model_dir + \"/inference.pdiparams\""
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "3d196913-6542-4177-87ab-c5aa1994f8e8",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Initialize the Paddle recognition inference on CPU\n",
+    "rec_ie = Core()\n",
+    "# read the model and corresponding weights from file\n",
+    "rec_net = rec_ie.read_model(model=rec_model_file_path, weights=rec_params_file_path)\n",
+    "\n",
+    "# assign dynamic shapes to every input layer on the last dimension\n",
+    "for input_layer in rec_net.inputs:\n",
+    "    input_shape = input_layer.partial_shape\n",
+    "    input_shape[3] = Dimension(-1)\n",
+    "    rec_net.reshape({input_layer: input_shape})\n",
+    "\n",
+    "rec_compiled_model = rec_ie.compile_model(model=rec_net, device_name=\"CPU\")\n",
+    "\n",
+    "# get input and output nodes\n",
+    "rec_input_layer = next(iter(rec_compiled_model.inputs))\n",
+    "rec_output_layer = next(iter(rec_compiled_model.outputs))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "573a1a11-faec-41af-bf43-08b90d28cec3",
+   "metadata": {},
+   "source": [
+    "### Preprocessing image functions for text detection and recognition"
```

Comment:
Could you include a markdown cell after this title, we can tell to the user what is the intention of the next two cells and why those are needed.

Reply:
Added a markdown cell for explanation